lib/Qi/Model/Base.pm

```diff
@@ -0,0 +1,35 @@
+package Qi::Model::Base;
+use strict;
+use warnings;
+use Mojo::Base 'Mojolicious';
```

Comment:
No need for strict and warnings, Mojo::Base gives that to you "for free".  "for free" is used to communicate that software does a good action for you on your behalf with no action on your point necessary.